code/modules/crafting/craft.dm

```diff
@@ -31,7 +31,7 @@
 /datum/personal_crafting/proc/get_environment(mob/user)
 	. = list()
 	. += range(2, user.loc)
-	. += user.contents
+	. += user.hands
 
```

Comment:
try "get_both_hands(user)". It returns a list of the objects in both of the users hands.